frontend/src/components/activity/content/StoryboardRow.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <component
+    :is="variant !== 'dense' ? 'tr' : 'div'"
```

Comment:
I agree as well. A11y components often are more complex. I assume you would want to extract the dense version?

I think this could be done in a improvement task. I believe, this UI improvement should land on stage for the closed beta, as it greatly improves the mobile view.